Mahmoud Mansour/css/styles.css

```
@@ -0,0 +1,154 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: Helvetica ,Arial, sans-serif;
+    box-sizing: border-box;
+}
+header {
+    width: 100%;
+    height: 65px;
+    box-shadow: 0px 0px 15px 0px #ccc;
+    z-index: 1px;
+    
+}
+
+.wrapper {
+    width: 800px;
+    margin: 0 auto;
+    
+}
+.logo img {
+    width: 40px;
+    height: auto;
+    margin-top: 5px;
+    float: left;
+   
+}
+.btns{
+    float: right;
+    margin: 10px;
+    margin-top: 22px;
+}
+.clr {
+    clear: both;
+}
+.btns a {
+    text-decoration: none;
+    border: 1px solid #33CCFF;
+    padding: 5px;
+    border-radius: 0.5em;
+    color: #33CCFF;
+    margin-right: 3px;
+}
+.btns a:nth-child(2) {
+    background-color: #33CCFF;
+    color: #fff;
+}
+.btns a:nth-child(2):hover {
+    background-color: rgb(1, 170, 255);
+    color: #fff;
+}
+.btns a:first-child:hover{
+    background-color:  rgb(1, 170, 255);
+    color: #fff;
+}
+.cover img{
+    width: 100%;
+    height: 350px;
+    overflow: hidden;
+}
+.profile-img img {
+    width: 135px;
+    height: 135px;
+    border-radius: 50%;
+    border: 2px solid #fff;
+    margin: -70px 10px -50px 25px;
+    box-shadow: 0px 0px 2px inset rgba(0, 0, 0, 0.02) ;
+}
+.profile-detalls { 
+    color: #232b2b;   
+    margin-left: 165px;
+    
+    
+}
+.profile-detalls h3 {
+    margin-bottom: 10px;
+    
+}
+.profile-detalls a {
+    color: #aaa;
+    text-decoration: none;
+}
+.profile-detalls a:visited {
+    color: #aaa;
+    text-decoration: none;
+}
+
+.icon {
+    margin-right: 8px;
+}
+.antrect-btns a {
+    text-decoration: none;
+    border: 1px solid #33CCFF;
+    padding: 5px;
+    border-radius: 0.5em;
+    color: #33CCFF;
+    margin-right: 3px;
+}
+.antrect-btns a:nth-child(2) {
+    background-color: #33CCFF;
+    color: #fff;
+}
+.antrect-btns a:nth-child(2):hover {
+    background-color: rgb(1, 170, 255);
+    color: #fff;
+}
+.antrect-btns a:first-child:hover {
+    background-color: rgb(1, 170, 255);
+    color: #fff;
+}
+.antrect-btns  {
+    text-align: right;
+    margin-top: -40px;
+    margin-right: 8px;
+}
+.spreate {
+    
+    margin-top: 50px;
+    margin-left: 55px;
+    margin-right: 55px;
+    
+    border: 1px solid #ddd;
+}
+.followers {
+    margin-left: 10px;
+    margin-top: 8px;
+    font-size: 0.90em;
+}
```

My only comment on you work is about the indentation and format
try to put new line between each two code sections and don't leave more than one there and don't leave new lines in random places in your code in order to make it more clean

```suggestion
}

.antrect-btns a:first-child:hover {
    background-color: rgb(1, 170, 255);
    color: #fff;
}

.antrect-btns  {
    text-align: right;
    margin-top: -40px;
    margin-right: 8px;
}

.spreate {    
    margin-top: 50px;
    margin-left: 55px;
    margin-right: 55px;    
    border: 1px solid #ddd;
}

.followers {
    margin-left: 10px;
    margin-top: 8px;
    font-size: 0.90em;
}

```